models/Card.js

```diff
@@ -40,9 +40,14 @@ Card.init(
       type: DataTypes.INTEGER,
       allowNull: false,
     },
-    e_factor: {
+    efactor: {
       type: DataTypes.FLOAT,
       allowNull: false,
+      field: 'e_factor',
+    },
+    dueDate: {
```

Comment:
Not sure if we need a `dueDate` property here. If it's related to when the card should be queued then that can be  calculated using the `interval` card property and `dayjs`.  There's an example in `update-queue.js` if you want to take a look. In PR #10

Reply:
Okay, that should work then! Yes, the `dueDate` property was just for comparing with the current date to test whether a card is due for review. I think that it's like that because the supermemo docs do not include a `is_queued` property and so they used this instead!